### DISCUSSION TITLE
Issue regarding https access-rules updation

### DISCUSSION THREAD

**Discussion by AbinDhiman123** [ory/oathkeeper#1057] [Q&A]
Hello, we are deploying the ory-oathkeeper to the kubernetes cluster. previously we used the access-rules.yml from /etc/config/oathkeeper/access-rules.yml and we deployed the ory-oathkeeeper to cluster and now we are using the https method for access-rules. If I am updating the access-rules using https, it needs re-deployment every time to reflect the changes what I made in the access-rules. how we can update the access-rules without re-deployment to the kubernetes cluster.

**Comment by jmls** [comment]
did you ever work this out ?
